Doc/library/functions.rst

```diff
@@ -43,8 +43,8 @@ are always available.  They are listed here in alphabetical order.
 .. function:: abs(x)
 
    Return the absolute value of a number.  The argument may be an
-   integer or a floating point number.  If the argument is a complex number, its
-   magnitude is returned.
+   integer or a floating point number, If *x* defines :meth:`__abs__`,
```

Comment:
There's a capital letter after a coma.
Also I have another proposition about this coma:

> [...] integer, a floating point number, a complex number, or any object defining the :meth:`__abs__` method.  If the argument is a complex number, its magnitude is returned.

I agree with @jdemeyer, it could be "simplified" even to ```Accept any object *x* implementing :meth:`__abs__` ```, but this documentation is not the language reference, its audience is newcomers, they care about being able to pass the three builtins they just learnt `int`, `float`, `complex` like `abs(-5)` and `abs(-5.5)`.